main.py

```
@@ -979,6 +987,12 @@ async def lounge_host(update: Update, context: ContextTypes.DEFAULT_TYPE):
                 return
                 
             if update.message.chat.type != "private":
+                if not ANTIGRAVITY_BYPASS_PASSWORD:
+                    try:
+                        await context.bot.send_message(chat_id=chat_id, text=ANTIGRAVITY_BYPASS_UNCONFIGURED_TEXT, parse_mode="HTML", reply_markup=CLOSE_KEYBOARD)
+                    except Exception as e: logging.debug(f"Ignored error: {e}")
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)